agent/util-scripts/pbench-tool-meister-start

```diff
@@ -395,8 +395,12 @@ def main(argv):
 
     # 3. Start the tool-data-sink process
     #   - leave a PID file for the tool data sink process
+    host_tools_dict = dict()
+    for host in tool_group.hostnames.keys():
+        host_tools_dict[host] = tool_group.get_tools(host)
+
     tds_param_key = "tds-{}".format(group)
-    tds = dict(channel=channel, benchmark_run_dir=benchmark_run_dir, group=group)
+    tds = dict(channel=channel, benchmark_run_dir=benchmark_run_dir, group=group, host_tools_dict=host_tools_dict)
```

Comment:
Why do we need this?

Reply:
We don't, it was part of the original work, I plan on removing it today.